src/services/codefixes/codeFixProvider.ts

```diff
@@ -0,0 +1,53 @@
+/* @internal */
+namespace ts {
+    export interface CodeFix {
+        name: string;

```

Comment:
If I'm not mistaken, `name` is unused. If so, then there's no real reason for it, right?